### DISCUSSION TITLE
Grain boundary velocity

### DISCUSSION THREAD

**Discussion by ashishdhole** [idaholab/moose#27437] [Q&A Modules: Phase field]
Hello, 
I am working on a friction pressure model of grain growth. and referring, I want to know if the velocity (_new_val) in grainboundaryvelosity auxkernel is same as what this paper mentioned in 

![Screenshot from 2024-04-22 10-48-12](https://github.com/idaholab/moose/assets/53962301/b469c570-6b91-46de-b0dd-0548ac79a75b)

without eta_top and eta_bottom condition?
Thank you

**Comment by ashishdhole** [comment]
The paper I am referring to is doi:10.1088/0965-0393/20/6/065008

**Comment by ashishdhole** [comment]
Also is there any way I can output the value of any variable at every time step. say for example I want to extract the value of _new_value at some quadrature point, is there any post processor or output module for it?

**Reply by GiudGiud** [reply]
Hello

what is new_value?
a postprocessor? a material property? a variable? a class attribute (of which class?)?

Guillaume

**Reply by ashishdhole** [reply]
Sorry my bad. In GrainBoundaryVelocity Auxkernel (https://github.com/idaholab/moose/blob/next/modules/phase_field/src/auxkernels/GrainBoundaryVelocity.C)

The class
GrainBoundaryVelocity::computeValue()

calculates velocity as _new_val. My question is is the _new_val same as the screenshot?

**Reply by GiudGiud** [reply]
it seems to be computed different, with 1 / grad_eta as opposed to d n / d eta but it seems to be intent
